sw/device/silicon_creator/lib/drivers/flash_ctrl.h

```diff
@@ -65,19 +65,19 @@ typedef enum flash_crtl_partition {
   /**
    * Select the data partition.
    */
-  kFlashCtrlRegionData = 0x0000,
+  kFlashCtrlPartitionData = 0,
   /**
    * Select the info partition of type 0.
    */
-  kFlashCtrlRegionInfo0 = 0x0100,
+  kFlashCtrlPartitionInfo0 = 1,
   /**
    * Select the info partition of type 1.
    */
-  kFlashCtrlRegionInfo1 = 0x0300,
+  kFlashCtrlPartitionInfo1 = 3,
```

Comment:
Might be worth spelling these out a bit more to make it clearer how the constant is composed.

```c
kFlashCtrlPartitionInfo0 = (0 << 1) | 1,
kFlashCtrlPartitionInfo1 = (1 << 1) | 1,
kFlashCtrlPartitionInfo2 = (2 << 1) | 1,
```

Reply:
Done.